src/components/connect-wallets-drawer/ConnectWalletsDrawer.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useCallback } from 'react'
+
+import MaskedView from '@react-native-masked-view/masked-view'
+import {
+  getModalVisibility,
+  setVisibility
+} from 'audius-client/src/common/store/ui/modals/slice'
+import { StyleSheet, View } from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+
+import Drawer from 'app/components/drawer'
+import Text from 'app/components/text'
+import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { ThemeColors, useThemeColors } from 'app/utils/theme'
+
+const MODAL_NAME = 'MobileConnectWalletsDrawer'
+
+const messages = {
+  title: 'Connect Wallets',
+  text:
+    'To connect additional wallets please visit audius.co from a desktop browser'
+}
+
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    container: {
+      paddingTop: 48,
+      paddingLeft: 16,
+      paddingRight: 16,
+      paddingBottom: 48
+    },
+
+    title: {
+      fontFamily: 'AvenirNextLTPro-Heavy',
```

Comment:
No need to change but you can set `weight="heavy"` on our `Text` component. I think it's a little cleaner than typing out the font family everywhere

Reply:
I agree. Used font weight so I don't have to write it twice though 👍

Reply:
Font-weight doesn't work on Android. Or iOS? one of the two doesn't work lol

Also I think eventually we should turn the masked text thing into a component so we don't have all this duplication everywhere